### PR TITLE
release Swish 2.6.0

### PR DESCRIPTION
**Pull request by owaddell**
- handle error returned by `osi_connect_tcp*`
- update to libuv 1.48.0
- update to SQLite 3.45.1
- remove obsolete porting documentation
- improve the `not-reached` macro

### CODE REVIEW

File: ChangeLog.md [hunk 1]
```diff
@@ -1,5 +1,74 @@
 # Change Log
 
+## 2024-03-11 Version 2.6.0 (Stable)
+
+### New
+
+* Enabled use with Chez Scheme 10.0
+* Enabled use with arm64osx, tarm64osx, and arm64le machine types
```
Comment (burgerrg) [inline ChangeLog.md]: arm64osx, tarm64osx, arm64le, and tarm64le
(I checked it on tarm64le.)

File: src/swish/io.ss [hunk 2]
```diff
@@ -1127,7 +1127,8 @@
             [(,who . ,errno) (io-error name who errno)]
             [,port
              (let ([name (osi-port-name port)])
-               (values (@make-iport name port #f) (@make-oport name port)))])]))))
+               (values (@make-iport name port #f) (@make-oport name port)))])]
+         [(,who . ,errno) (io-error name who errno)]))))
```
Comment (burgerrg) [inline src/swish/io.ss:1131]: Good find!

**Review by burgerrg — APPROVED** [review]

**Review by burgerrg — APPROVED** [review]
Thanks for this significant update to Swish!

**Comment by owaddell** [comment]
Thanks for all your work in reviewing this release, Bob!

**Pull request merged**